integration/nwo/network.go

```diff
@@ -184,6 +185,7 @@ func New(c *Config, rootDir string, client *docker.Client, startPort int, compon
 		PortsByBrokerID:   map[string]Ports{},
 		PortsByOrdererID:  map[string]Ports{},
 		PortsByPeerID:     map[string]Ports{},
+		BootstrapMethod:   "file",
```

Comment:
Why is this at the top level and not under 'consensus' or channel?

Reply:
Done